### ISSUE TITLE
Scanning Area

### ISSUE THREAD

**Issue by HoGDylan** [JimmyPun610/BarcodeScanner.Mobile#166]
Hello

I was wondering if there is a way to change the scanning area? Let's say I have a page with a list of barcodes, I would like a sort of frame on screen where the actual scanning happens so it's possible to "aim" an choose wich one I scan want to scan.

**Comment by JimmyPun610** [comment]
currently no, should be related to #129 
It is facing a critical problem that the bounding is returning native coordination and the coordination is not precise.

**Comment by pulmuone** [comment]
https://github.com/tmoncorp/Xamarin.Barcode
good luck

**Comment by scriptBoris** [comment]
> https://github.com/tmoncorp/Xamarin.Barcode удачи

No undestand

**Comment by adrianotrentim** [comment]
Any solutions to this?

**Comment by UkeHa** [comment]
Yes, using this https://github.com/afriscic/BarcodeScanning.Native.Maui repo it works out of the box
